frontend/src/features/help/components/text/CreateDisposalNavigationHelpText.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+/**
+ * Help Text for the creation of disposal project.
+ */
+export const CreateDisposalNavigationHelpText = () => {
+  return (
+    <p>
+      A stepper is provided to guide you through the disposal process. You will not be able to
+      advance to the next step until validation has passed for the current step.
+    </p>
```

Comment:
I know this is just a first pass so not a huge deal, but I don't think non-tech-y people will quite understand what "validation" entails

Reply:
Kinda had that thought in my head too..